examples/stress_vcm/in.stress_vcm.lmp

```diff
@@ -0,0 +1,113 @@
+# Removing Binned Velocities of Center of Mass (VCM) from Stress 
```

Comment:
None of the input scripts in the examples lower-case dirs have a *.lmp suffix - so just in.stress_vcm

Reply:
fixed